src/pages/Spacing.vue

```diff
@@ -0,0 +1,304 @@
+<script setup>
+// Note there's a way we can just import these globally so we don't have to do it in every page
+import Box from '../components/Box.vue'
+import Container from '../components/Container.vue'
+import WidthController from '../components/WidthController.vue'
+
+const exampleClasses = 'p-4 rounded-2 text-white flex items-center justify-center'
+</script>
+<style>
+.bg-stripes-white {
+  background-image: linear-gradient(135deg,hsla(0,0%,100%,.75) 10%,transparent 0,transparent 50%,hsla(0,0%,100%,.75) 0,hsla(0,0%,100%,.75) 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+.bg-stripes-purple {
```

Comment:
You _should_ be able to re-use Box to do these stripes, it knows how to do stripes or dots, and also how to make those different colors.

Example for purple:

```
<box striped fg-color="#a855f780" bg-color="#c084fc1a" />
```